share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/profile/userprofile.get.properties

```diff
@@ -9,3 +9,4 @@ label.photoimagesize=Your image will be resized to 64px by 64px.
 label.photonote=Changes to your photo image take effect immediately.
 button.editprofile=Edit Profile
 button.usedefault=Use Default
+label.restricted-settings=Some settings are restricted by your organization.
```

Comment:
@alexbalmus Can we keep the label.restricted-settings option only in one value, set it in a globally scoped properties file.